client/my-sites/customer-home/locations/secondary/index.jsx

```diff
@@ -23,16 +26,39 @@ const cardComponents = {
 };
 
 const Secondary = ( { cards } ) => {
+	if ( ! cards ) {
+		return null;
+	}
+	const allPossibleEducationalCards = [
+		'home-education-gutenberg',
+		'home-education-free-photo-library',
+	];
+	const educationalCards = cards.filter( card => allPossibleEducationalCards.includes( card ) );
+	let skipEducationalCard = false;
 	return (
 		<>
-			{ cards &&
-				cards.map(
-					( card, index ) =>
-						cardComponents[ card ] &&
-						React.createElement( cardComponents[ card ], {
-							key: index,
-						} )
-				) }
+			{ cards.map( ( card, index ) => {
+				if ( ! cardComponents[ card ] ) {
+					return null;
+				}
+				if ( educationalCards.includes( card ) ) {
+					if ( skipEducationalCard ) {
+						return null;
+					}
+
+					skipEducationalCard = true;
```

Comment:
I admit this kinda feels like a step backwards to me: handling certain cards a certain way in a certain location in the client 🤔. I feel like if we're really sticking to our "client as simple as possible" maxim, the location components should basic look the exact same, and rarely get changed.

I'm wondering if there's a way to annotate DotPager collections in the API response, so that location components would:
* pulls card groups and renders them with a DotPager,
* renders remaining single cards.

IIRC with some mockups, we're likely need this for at least the secondary and tertiary locations anyways.

Reply:
> I'm wondering if there's a way to annotate DotPager collections in the API response, so that location components would:

Yup the experimental view does something pretty similar here. Basically we just need to update our current views as well. Something like: 
```
const SITE_ESTABLISHED_DESKTOP = [
	'notices' => [
		NOTICE_CELEBRATE_SITE_MIGRATION,
		NOTICE_CELEBRATE_SITE_LAUNCH,
		NOTICE_CELEBRATE_SITE_SETUP_COMPLETE,
	],
	'upsells' => [
		BANNER_LEGACY_STATS_BANNERS,
	],
	'primary' => [
		PRIMARY_QUICK_LINKS,
	],
        'secondary.educational-pager' => [
            EDUCATION_GUTENBERG,
            EDUCATION_FREE_PHOTO_LIBRARY,
       ],
	'secondary' => [
		FEATURE_STATS,
		FEATURE_SUPPORT,
                 EDUCATIONAL_PAGER,
                	FEATURE_GROW_EARN,
		FEATURE_GO_MOBILE_DESKTOP,
	],
	'view_name' => 'SITE_ESTABLISHED_DESKTOP',
];
```

Reply:
> I admit this kinda feels like a step backwards to me

you're not alone! 😄 This should be considered a temporary hack in order to display the Mastering Gutenberg card while we ship the new Home layout. I've should have put a huge warning here setting the expectations of this hack.

As @gwwar pointed out, the experimental views we added to the API already support this and I played around with it in https://github.com/Automattic/wp-calypso/pull/41000 (see b806c10 or more specifically https://github.com/Automattic/wp-calypso/pull/41000/commits/b806c106fae44beb4725b7d8c479c14004f147c1#diff-f718bf14b2a68d4096184913827aca05R30)

Reply:
If we land with a hack, let's maybe add a cleanup issue if for some reason we're not able to iterate quickly enough to turn on the experimental layouts.